integration/podman/podman.go

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) 2020 Intel Corporation
```

Comment:
this file is very similar to `docker.go`, I think we should factorize it later

Reply:
@devimc yeah, I was trying to do that maybe in another PR, I will do it (the issue that I have currently is that I need to identify which things are working with podman and which are not, some commands can't be used rootless)